### ISSUE TITLE
Update to 3.7 lizmap version

### ISSUE THREAD

**Issue by Alain-LG** [NaturalGIS/qgis_server_and_lizmap_on_windows#9]
Hello,

I use your tutorial which is very useful and it's ok to use the 3.6 lizmap version.
However, i would like to try the new version (3.7). I follow the lizmap documentation to upgrade it but i have many errors with this message : "Error 500. A technical error has occured. Sorry for this trouble." when i am connected as administrator.

When i am not connected, the map begin to open but the only thing which appears is the waiting picture.

Do you know how to resolve these problems please or when do you think to update your script ?

Thanks a lot for your help

**Comment by gioman** [comment]
> Do you know how to resolve these problems please or when do you think to update your script ?

@Alain-LG we have not yest considered or tested upgrading this package for windows in order to use LMWC 3.7. From our preliminary tests on Linux LMWC 3.7 has a few issues (see the LMWC bug tracker) that do not make it ready yet for prime time. Anyway usually the guys from 3Liz are very active and I expect that those issue will be solved soon, if not already.

**Comment by nboisteault** [comment]
Lizmap 3.7.1 is out! :)

**Comment by sj250irff** [comment]
Bonjour,

-Pensez-vous que la version master est compatible avec windows 11 ? -Apache ne s'installe pas. j'avais déjà installé une version sur windows 10 et tout fonctionnait.

Merci
Cordialement

**Comment by nboisteault** [comment]
Bonjour,
Si vous souhaitez tester Lizmap, vous pouvez le faire avec docker en suivant les instructions de :

- https://github.com/3liz/lizmap-docker-compose
ou
- https://github.com/3liz/lizmap-web-client/tree/master/tests#testing-lizmap
ou encore
- Github codespaces directement depuis https://github.com/3liz/lizmap-web-client sans rien installer sur votre poste

**Comment by gioman** [comment]
> Bonjour, Si vous souhaitez tester Lizmap, vous pouvez le faire avec docker en suivant les instructions de :
> 
> * https://github.com/3liz/lizmap-docker-compose
>   ou
> * https://github.com/3liz/lizmap-web-client/tree/master/tests#testing-lizmap
>   ou encore
> * Github codespaces directement depuis https://github.com/3liz/lizmap-web-client sans rien installer sur votre poste

@nboisteault I agree, docker or any other installation method recipe on a linux server should be used for testing or production, as qgis server is really not meant to be run on Windows. That said... docker and/or Linux are not something that anyone can handle. What motivated us in the first place to have a recipe for Windows is that there are many GIS users/technicians that are wanting to try QGIS Server/Lizmap but don't have the necessary IT skills to handle docker and/or linux. 

@sj250irff this repo is not updated because for recent Lizmap releases is mandatory the use of a specific QGIS Server / Lizmap plugin and the last time I tried I was not able to make Apache / QGIS Server on Windows to read / use QGIS Server plugins. Possibly I was just doing something wrong, but since I had no more time to try again.
